packages/components/src/alignment-matrix-control/utils.tsx

```diff
@@ -68,10 +99,16 @@ export function getItemId(
  * @return The index of a matching alignment.
  */
 export function getAlignmentIndex(
-	alignment: AlignmentMatrixControlValue = 'center'
+	alignment: VoidableAlignmentMatrixControlValue = 'center'
 ) {
-	const item = transformValue( alignment );
-	const index = ALIGNMENTS.indexOf( item );
+	const transformedValue = transformValue(
+		alignment
+	) as AlignmentMatrixControlValue;
```

Comment:
Instead of casting, can we add better support for the nullish values we decide to support? Seems like we should be able to just return `undefined` if `transformedValue` is nullish itself.